ui/page/livestream/view.jsx

```diff
@@ -163,12 +165,9 @@ export default function LivestreamPage(props: Props) {
   }, [uri, stringifiedClaim, isAuthenticated, doUserSetReferrer]);
 
   React.useEffect(() => {
-    if (!layountRendered) return;
-
     doSetPrimaryUri(uri);
-
     return () => doSetPrimaryUri(null);
-  }, [doSetPrimaryUri, layountRendered, uri]);
+  }, [doSetPrimaryUri, layountRendered, uri, isStreamPlaying]);
```

Comment:
- Is `layoutRendered` still needed in the dependency after the removal?

Reply:
Probably not. The removal was experimental in the beginning because I wasn't 100% sure why it's there to begin with. It was the cause for some resizing issues though. I'll remove the dependency.